public/video-ui/src/pages/Video/index.jsx

```diff
@@ -372,7 +372,7 @@ class VideoDisplay extends React.Component {
 
   render() {
     const video = this.props.video &&
-      this.props.params.id === this.props.video.id
+      (this.props.params.id === this.props.video.id || this.state.isCreateMode)
```

Comment:
This is necessary as we have made the video `id` attribute a blank string, causing it to no longer match on the create view.